docs/source/developer/dev.rst

```diff
@@ -0,0 +1,35 @@
+Dev Application
+##############################################################################
+
+This application:
+
+1. Provides a quick reference for seeing what content has been added to the databse.
+2. Allows authors to check markdown content has rendered as expected.
+3. Is to be used in a development environment.
+
+This application is available only when ``DEBUG`` mode is enabled (and therefore is not
+visible in production).
+
+This application does not necessarily show the same connection/layout of content as in
+the actual site itself. This is because some of the links in this app link to the actual
+site, but many link to a copy of the template from the topics app (and are therefore are
+likely to be out of date at any given point in time).
+
+
+What does this app contain?
```

Comment:
On second thoughts, we could keep the title but not go into this much detail in this section, for example:

```
What does this app contain?
========================================

The dev application currently contains the following:

- A webpage displaying details of all content for the topics application.
```